tgui/packages/tgui/interfaces/Pressurizer.js

```diff
@@ -0,0 +1,223 @@
+                onClick={() => handleSetFan(FanState.In)}
+              >
+                In
+              </Button>
+              <Button
+                color={fanState === FanState.Out ? 'good' : 'default'}
+                onClick={() => handleSetFan(FanState.Out)}
+              >
+                Out
+              </Button>
+            </LabeledList.Item>
+            <LabeledList.Item label="Release Pressure">
+              <Button onClick={() => handleSetPressure(minRelease)}>
+                Min
+              </Button>
+              <NumberInput
+                animated
+                width="7em"
+                value={releasePressure}
+                minValue={minRelease}
+                maxValue={maxRelease}
+                onChange={(e, targetPressure) => handleSetPressure(targetPressure)} />
+              <Button onClick={() => handleSetPressure(maxRelease)}>
+                Max
+              </Button>
+            </LabeledList.Item>
+          </LabeledList>
+        </PortableBasicInfo>
+        <Section
+          title="Material Processing"
+          buttons={(
+            <Button
+              icon="eject"
+              disabled={!materialsCount}
+              onClick={() => handleEjectContents()}>
+              Eject
+            </Button>
+          )}>
+          <LabeledList>
+            <LabeledList.Item label="Speed">
+              <Button
+                color={processRate === 1 ? 'good' : 'default'}
+                onClick={() => handleSetProcessRate(1)}
+              >
+                1
+              </Button>
+              <Button
+                color={processRate === 2 ? 'good' : 'default'}
+                onClick={() => handleSetProcessRate(2)}
+              >
+                2
+              </Button>
+              <Button
+                color={processRate === 3 ? 'good' : 'default'}
+                onClick={() => handleSetProcessRate(3)}
+              >
+                3
+              </Button>
+              {!!emagged && (
+                <Button
+                  color={processRate === 4 ? 'good' : 'default'}
+                  onClick={() => handleSetProcessRate(4)}
+                >
+                  4
+                </Button>
+              )}
+              {!!emagged && (
+                <Button
+                  color={processRate === 5 ? 'good' : 'default'}
+                  onClick={() => handleSetProcessRate(5)}
+                >
+                  5
+                </Button>
+              )}
+            </LabeledList.Item>
+            <LabeledList.Item label="Progress">
+              <ProgressBar
+                mt="0.5em"
+                ranges={{
+                  good: [1, Infinity],
+                  average: [0.75, 1],
+                  bad: [-Infinity, 0.75],
+                }}
+                value={materialsProgress/100} />
+            </LabeledList.Item>
```

Comment:
Based on the (maybe out of date) screenshot this displays a little weirdly. Could we get some up-to-date UI screenshots in the ticket? II think you can get rid of `mt="0.5em"` here and rely on the `LabeledList.Item`s for layout.